### PR TITLE
Adding omri-sass

### PR DESCRIPTION
**Pull request by omris1**
Second attempt at this, now with my proper name, a standard filename and a respective record in the README.md file. Hoping I did the rebase thing properly.

### CODE REVIEW

File: people/omri-sass.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Omri Sass
+-------------
+
+![](photos/OmriSass.png)
+
```
Comment (dblock) [inline people/omri-sass.md]: Rename the png file to match as well, please.

**Comment by dblock** [comment]
You rebased correctly but you still will have to squash these commits into one after you've fixed the file name for the jpg.

**Comment by omris1** [comment]
Hope this is right...

**Comment by dblock** [comment]
Nope, look at all these files in the diff that aren't yours :) Find me on slack and I'll help you?

**Pull request closed without merging**